README.md

```diff
@@ -1 +1,16 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
```

Comment:
- [ ] [Required] I kindly suggest that you add a description of the project underneath this section

**EXAMPLE**
```
# Hello Microverse
This is my first Microverse project
```